eng/pipelines/templates/stages/vmr-build.yml

```diff
@@ -81,7 +81,7 @@ stages:
         demands: ${{ variables.defaultPoolDemands }}
       container: ${{ parameters.centOSStream8Container }}
       buildFromArchive: false            # 🚫
-      enablePoison: false                # 🚫
+      enablePoison: true                 # ✅
```

Comment:
Reminder to revert

Reply:
Reverted in https://github.com/dotnet/installer/pull/17605/commits/992274661254c76c17ee15b227a5f7fbac951c4a